.ci/e2e-batching/ddl-batch.txt

```diff
@@ -0,0 +1 @@
+CREATE TABLE table1 (id integer PRIMARY KEY, data integer); CREATE TABLE table2 (id integer PRIMARY KEY, data integer); CREATE TABLE table3 (id integer PRIMARY KEY, data integer);
```

Comment:
Can we also add some negative tests? e.g: SELECT statements in batch, DML + DDL used together and see what errors we return?

Reply:
I added additional tests.